specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2023-11-01/SourceControls.json

```diff
@@ -0,0 +1,1005 @@
+        },
+        "deploymentLogsUrl": {
+          "description": "Url to access repository action logs.",
+          "type": "string"
+        }
+      }
+    },
+    "ContentType": {
+      "description": "The content type of a source control path.",
+      "enum": [
+        "AnalyticRule",
+        "AutomationRule",
+        "HuntingQuery",
+        "Parser",
+        "Playbook",
+        "Workbook"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "ContentType",
+        "values": [
+          {
+            "value": "AnalyticRule"
+          },
+          {
+            "value": "AutomationRule"
+          },
+          {
+            "value": "HuntingQuery"
+          },
+          {
+            "value": "Parser"
+          },
+          {
+            "value": "Playbook"
+          },
+          {
+            "value": "Workbook"
+          }
+        ]
+      }
+    },
+    "RepoType": {
+      "description": "The type of repository.",
+      "enum": [
+        "Github",
+        "AzureDevOps"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "RepoType",
+        "values": [
+          {
+            "value": "Github"
+          },
+          {
+            "value": "AzureDevOps"
+          }
+        ]
+      }
+    },
+    "RepositoryAccessKind": {
+      "description": "The kind of repository access credentials",
+      "enum": [
+        "OAuth",
+        "PAT",
+        "App"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "RepositoryAccessKind",
+        "values": [
+          {
+            "value": "OAuth"
+          },
+          {
+            "value": "PAT"
+          },
+          {
+            "value": "App"
+          }
+        ]
+      }
+    },
+    "Version": {
+      "readOnly": true,
+      "description": "The version of the source control.",
+      "enum": [
+        "V1",
+        "V2"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "Version",
+        "values": [
+          {
+            "value": "V1"
+          },
+          {
+            "value": "V2"
+          }
+        ]
+      }
+    },
+    "DeploymentFetchStatus": {
+      "description": "Status while trying to fetch the deployment information.",
+      "enum": [
+        "Success",
+        "Unauthorized",
+        "NotFound"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "DeploymentFetchStatus",
+        "values": [
+          {
+            "value": "Success"
+          },
+          {
+            "value": "Unauthorized"
+          },
+          {
+            "value": "NotFound"
+          }
+        ]
+      }
+    },
+    "DeploymentState": {
+      "description": "The current state of the deployment.",
+      "enum": [
+        "In_Progress",
```

Comment:
InProgress (without underscore)?

Reply:
This is the value provided by external system and we're doing passthrough.